packages/@apphosting/create/src/bin/create.ts

```diff
@@ -1,21 +1,49 @@
 #! /usr/bin/env node
-import { spawn } from "child_process";
+import { spawnSync, spawn } from "child_process";
 import { program } from "commander";
+import { downloadTemplate } from "giget";
+import { select } from "@inquirer/prompts";
 
 program
   .option("--framework <string>")
   .argument("<directory>", "path to the project's root directory")
-  .action((directory, { framework }: { framework?: string }) => {
+  .action(async (dir, { framework }: { framework?: string }) => {
+    // TODO validate the framework
     if (!framework) {
-      throw new Error(
-        "Framework selecter not implemented. Must provide an option to --framework <string>",
-      );
+      framework = await select({
+        message: "Select a framework",
+        choices: [
+          { name: "Angular", value: "angular" },
+          { name: "Next.js", value: "nextjs" },
+        ],
+      });
     }
-    const adapterName = `@apphosting/adapter-${framework}`;
-    const buildCommand = `apphosting-adapter-${framework}-create`;
-    spawn("npx", ["-y", "-p", adapterName, buildCommand, directory], {
+    // TODO allow different templates
```

Comment:
Templates besides the 2 boilerplates can be a fast-follow. The FriendlyEats/Chat codelabs won't rely on this script